docs/QUEST_SYSTEM.md

```
@@ -1,269 +1,115 @@
-      {activeCluster.sides.map(q => (
-        <QuestItem key={q.id} quest={q} onComplete={completeQuest} />
-      ))}
-    </div>
-  );
-};
-```
-
----
-
-*Every quest traces to a seed. Reproducible, testable, balanced.*
+```
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)